modules/etcd-node-scaling.adoc

```diff
@@ -6,11 +6,11 @@
 [id="etcd-node-scaling_{context}"]
 = Node scaling for etcd
 
-In general, clusters must have 3 control plane nodes. However, if your cluster is installed on a bare metal platform, you can scale a cluster up to 5 nodes as a post-installation task. For example, to scale from 3 to 4 nodes after installation, you can add a host and install it as a control plane node. Then, the etcd Operator scales accordingly to account for the additional node.
+In general, clusters must have 3 control plane nodes. However, if your cluster is installed on a bare metal platform, you can scale a cluster up to 5 control plane nodes as a post-installation task. For example, to scale from 3 to 4 control plane nodes after installation, you can add a host and install it as a control plane node. Then, the etcd Operator scales accordingly to account for the additional control plane node.
```

Comment:
Out of scope to your PR, and I can't even find a style guide ruling on this, but I know that our [post install section](https://docs.openshift.com/container-platform/4.17/post_installation_configuration/index.html) of the docs was somewhat recently reworded to use "postinstallation" as a single word. I don't like it personally, but changing it here would make it more consistent with our docs. Up to you:
```suggestion
In general, clusters must have 3 control plane nodes. However, if your cluster is installed on a bare metal platform, you can scale a cluster up to 5 control plane nodes as a postinstallation task. For example, to scale from 3 to 4 control plane nodes after installation, you can add a host and install it as a control plane node. Then, the etcd Operator scales accordingly to account for the additional control plane node.
```

Reply:
Good to know! I'll fix that.